src/entities/Playlists/Playlist.js

```diff
@@ -106,6 +106,28 @@ class Playlist extends Entity {
     return new Playlist(playlist);
   }
 
+  /**
+   * Edit the playlist
+   * @param {String} newName - New name for the playlist
+   * @param {String} phraseList - New name for the playlist
+   * @returns {Playlist} true if successed
+   */
+  static async edit(playlist, newName, phraseList) {
+    if (!playlist.id || !newName) return;
+    try {
```

Comment:
Must be implemented in dva model

Reply:
Other function including edit are part of the class Playlist. I think implementing this and all the other function should be a separate issue. Because It would need a lot of refactoring